static/sass/main.scss

```diff
@@ -120,24 +121,20 @@ header {
   nav {
     display: flex;
     align-items: center;
-    margin-left: $content-padding;
+    margin: 0 $content-padding;
     -webkit-font-smoothing: initial;
 
     a {
-      @include gradient-bar-bg;
       cursor: pointer;
       font-size: 16px;
       text-align: center;
-      border-radius: $default-border-radius;
-      border-bottom: 1px solid $bar-border-color;
-      border-right: 1px solid $bar-border-color;
+      padding: 8px 16px;
+      color: $nav-link-color;
       white-space: nowrap;
 
-      &:active {
-        position: relative;
-        top: 1px;
-        left: 1px;
-        box-shadow: 3px 3px 4px $bar-shadow-color;
+      &:hover {
+	color: black;
```

Comment:
nit: indentation

Reply:
This was a tab.  I replaced it with spaces.